src/gui/log/logfiltermodel.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2019  sledgehammer999 <hammered999@gmail.com>
```

Comment:
@sledgehammer999:  **Side Question:** 

In previous PRs, most instances of this e-mail were replaced with `sledgehammer999@qbittorrent.org`.  Is there a particular reason you are switching back to your Gmail in this PR?

Reply:
I remember that I explicitly reverted that PR at some point. I hope someone didn't do it again.

The qbittorrent.org email is forwarded to the gmail one. If at some point in the future the qbittorrent.org email goes down or is blocked (eg I leave and the new admin won't redirect my emails), I want people to be able to contact me.